lib/smart_answer/calculators/benefit_cap_calculator_data_query.rb

```diff
@@ -1,8 +1,28 @@
 module SmartAnswer::Calculators
   class BenefitCapCalculatorDataQuery
-    attr_reader :data
+    attr_reader :data, :benefits, :questions
     def data
       @data ||= YAML.load_file(Rails.root.join('lib', 'data', 'benefit_cap_data.yml'))
     end
+
+    def benefits
```

Comment:
I'd prefer to see the addition of this method combined with the next commit ("Update the benefit cap flow to use the data convenience methods" - 54b51dfbd0d560cf0be68a265733fee12c57d758) to make it clearer how it's being used.